test/test_ops.py

```diff
@@ -652,6 +652,97 @@ def test_noncontiguous_samples(self, device, dtype, op):
             for i, (t, n) in enumerate(zip(t_grads, n_grads)):
                 self.assertEqual(t, n, msg=msg.format(i))
 
+    @unittest.skipIf(not torch._C._enable_cow_instrumentation(), "copy on write not instrumented")
+    @onlyNativeDeviceTypes
+    @suppress_warnings
+    @ops(op_db, allowed_dtypes=(torch.float32, torch.long, torch.complex64))
+    def test_lazy_copy_samples(self, device, dtype, op):
```

Comment:
I'm not... sure what you are intending to test here? The big ask is to check that view operations preserve non-standard shadow storage. This test is testing that every operator gives the same result if the storage has shadow storage vs not but... like, why would we expect this not to hold? We didn't change any of the Storage logic itself.

Reply:
In other words, this test is fairly expensive, seems to expose flakiness (e.g., cov), and doesn't even seem to test what is likely to be wrong. It doesn't seem like good ROI.